modules/ROOT/pages/amazon/amazon-dynamodb-connector-examples.adoc

```diff
@@ -0,0 +1,700 @@
+|Region Endpoint |Region for the Amazon DynamoDB client |`us-east-1`
+|===
+
+The following screenshot shows an example of configuring a global element for Amazon DynamoDB Connector:
+
+.Configure the credentials needed to access Amazon DynamoDB in the *Connection* section of the global element.
+image::amazon/amazon-dynamodb-global-element.png[Amazon DynamoDB global element]
+--
++
+. Click *Test Connection* to confirm that Mule can connect with the Amazon DynamoDB instance:
+* If the connection is successful, click *OK* to save the configuration.
+* If the connection is unsuccessful, review or correct any incorrect parameters, and then test again.
+. Click *OK*.
+. In the Create table properties window, configure the following fields:
++
+[%header,cols="25,35,40a"]
+|===
+|Field |Description | Value
+|Table name | The name of the table | Enter `Issues`.
+|Attribute definitions | An array of attributes that describe the key schema for the table and its indexes | Select `Edit inline`, click the green plus sign (+), and add the following attributes:
+
+* `IssueId` as type `STRING`
+* `Title` as type `STRING`
+* `CreateDate` as type `STRING`
+* `DueDate` as type `STRING`
+
+|Key schemas | The attributes that make up the primary key for the table or one of its indexes | Select `Edit inline`, click the green plus sign (+), add the following attributes:
+
+* `IssueId` as key type `HASH`
+* `Title` as key type `RANGE`
+|Read capacity units | The maximum number of strongly consistent reads consumed per second before Amazon DynamoDB returns a throttling exception | Enter `5`.
+|Write capacity units | The maximum number of writes consumed per second before Amazon DynamoDB returns a throttling exception | Enter `5`.
+|Global secondary indexes | An optional, additional key structure for the table | Enter `#[vars.secondary]` to create a variable for the secondary index, and then add the secondary index in the XML code. For information, see <<secondary-index, Add a Secondary Index to the Issues Table>>.
+|===
++
+The following screenshot shows an example of configuring the `Create table` operation:
++
+.Enter values for the `Create table` operation fields.
+image::amazon/amazon-dynamodb-create-table-props.png[Create table operation fields]
++
+In this screenshot:
+
+* The first item specifies the name of the component that represents the `Create table` operation in the flow.
+* The second item specifies the name of the table to create.
+* The third item defines the attributes of the new table.
+* The fourth item defines the key schemas of the new table.
+* The fifth items specify the maximum number of strongly consistent reads consumed per second and strongly consistent writes consumed per second before Amazon DynamoDB returns a throttling exception.
+* The sixth item species a variable for the secondary index.
+
+The XML flow for the Create Table example should look like this after you configure the `HTTP Listener` and `Create table` operations:
```

Comment:
Not sure we should capitalize Create Table, since we never introduced it as a name....  GLOBAL